tests/framework/resourcemanager.go

```diff
@@ -311,3 +323,86 @@ func (rm *ResourceManager) waitForRoutesToBeReady(ctx context.Context, namespace
 		},
 	)
 }
+
+// GetLBIPAddress gets the IP or Hostname from the Loadbalancer service.
+func (rm *ResourceManager) GetLBIPAddress(namespace string) (string, error) {
+	ctx, cancel := context.WithTimeout(context.Background(), rm.TimeoutConfig.CreateTimeout)
+	defer cancel()
```

Comment:
nit: a newline after defer statements help with readability
```suggestion
	defer cancel()
	
```